### PR TITLE
Blender mapping

### PR DESCRIPTION
**Pull request by un-tone**
In scope of https://github.com/Starcounter/CompanyTrack/issues/292

### CODE REVIEW

File: src/Launcher/Api/Map.cs [hunk 1]
```diff
@@ -6,12 +6,12 @@ public class MapHandlers
...
-            Blender.MapUri("/launcher/app-name", "/app-name");
-            Blender.MapUri("/launcher/partials/settings", "/settings");
-            Blender.MapUri("/launcher/partials/dashboard", "/dashboard");
-            Blender.MapUri("/launcher/partials/search?query={?}", "/search?query={?}");
+            Blender.MapUri("/launcher/user", "user");
+            Blender.MapUri("/launcher/menu", "menu");
+            Blender.MapUri("/launcher/app-name", "app-name");
+            Blender.MapUri("/launcher/partials/settings", "settings");
+            Blender.MapUri("/launcher/partials/dashboard", "dashboard");
+            Blender.MapUri("/launcher/partials/search?query={?}", "search");
```
Comment (warpech) [inline src/Launcher/Api/Map.cs]: So, does searching actually work with the above config?

Comment (un-tone) [inline src/Launcher/Api/Map.cs]: No, and it doesn't work with this changes https://github.com/StarcounterApps/Launcher/pull/286

Comment (warpech) [inline src/Launcher/Api/Map.cs]: Could you pls pull current `develop` to `blender-mapping` so we can continue investigating?

Comment (un-tone) [inline src/Launcher/Api/Map.cs]: Done.

**Pull request merged**